### PR TITLE
Content and Flow changes to getting started

### PR DESCRIPTION
**Pull request by ardan-bkennedy**

### CODE REVIEW

File: content/examples/getting-started.md [hunk 1]
```diff
@@ -1,144 +1,242 @@
...
-This document is a guide to introduce people to the `gb` project structure. A `gb` project is conceptually a `$GOPATH` per project, but saying that doesn't really help explain how to set up a new project; hence this document.
+This document is a guide to show you how to structure `gb` based Go projects and use `gb` to build those projects.
 
-# Creating an empty project
+A typical `gb` project may look like this on disk:
 
-A `gb` project is defined as any directory that has a `src/` subdirectory. The simplest possible `gb` project would be
+    /home/dfc/projects/engine/
+                        |——— src/
+                        |——— cmd/
```
Comment (davecheney) [inline content/examples/getting-started.md:14]: this is wrong, `cmd/` should live under `src/`

File: content/examples/getting-started.md [hunk 2]
```diff
@@ -1,144 +1,242 @@
...
+                              |——— engine/
+                                    |——— main.go
+                        |——— data/
```
Comment (davecheney) [inline content/examples/getting-started.md:17]: for the sake of clarity please remove data/ and services/

File: content/examples/getting-started.md [hunk 3]
```diff
@@ -1,144 +1,242 @@
...
+                        |——— services/
+                        |——— vendor/
+                              |——— src/
+                                    |——— github.com/codegangsta/cli/
+
+Since the `gb` tool can identify the workspace location of the project, imports can be referenced based on their location inside the `src/` and `vendor/src/` directories:
+
+    import (
+        "data"
```
Comment (davecheney) [inline content/examples/getting-started.md:26]: if these are packages, not folders at the root of the project, please fix the example output above

File: content/examples/getting-started.md [hunk 4]
```diff
@@ -1,144 +1,242 @@
...
-- The package name `all` matches all the packages inside your project's `src/` directory. It's a simple way to build everything, you can use other import paths and globs.
-- There is no way to build your vendored source, it will be built if required to build your code in the `src/` directory.
-
-## More complicated example
-
-For the second example we'll take a project that uses `godep` vendoring and convert it to be a `gb` project. First we'll need to setup a project and get the source
+With the code inside the `gb` project, we can now build the code. We are using the `all` option to have `gb` build all the packages inside the project:
+
+     % gb build all
+     2015/04/29 13:39:44 INFO project root "/home/dfc/devel/sftp"
```
Comment (davecheney) [inline content/examples/getting-started.md:114]: this is the old output, please redo the example with the updated output.

File: content/examples/getting-started.md [hunk 5]
```diff
@@ -1,144 +1,242 @@
...
+     2015/04/29 13:39:44 INFO build duration: 486.967µs map[]
+     2015/04/29 13:39:44 command "build" failed: failed to resolve package "github.com/pkg/sftp": cannot find package "github.com/kr/fs" in any of:
+             /home/dfc/go/src/github.com/kr/fs (from $GOROOT)
+             /home/dfc/devel/sftp/src/github.com/kr/fs (from $GOPATH)
+             /home/dfc/devel/sftp/vendor/src/github.com/kr/fs
+
+When we build the project, it fails because we are missing some dependencies the `gb` tool could not locate. When this happens, we can use the `vendor` command to fetch the dependencies and vendor them inside the project:
+
+     % gb vendor github.com/kr/fs
```
Comment (davecheney) [inline content/examples/getting-started.md:123]: i'm sorry but I don't want to have a reference to gb-vendor inside the main documentation. After talking to jason from google and others we decided that gb-vendor, being an obvious wrapper around go get was confusing as gb makes a big point of saying it does _not_ wrap go get. There is a page in plugins/gb-vendor where this example can move to to show an alternative way to initially set up the project

File: content/examples/getting-started.md [hunk 6]
```diff
@@ -1,144 +1,242 @@
...
+     2015/04/29 13:42:02 INFO project root "/home/dfc/devel/sftp"
```
Comment (davecheney) [inline content/examples/getting-started.md:124]: everything from here down is out of date, please consider the comment above.

**Comment by davecheney** [comment]
Thanks Bill. There are a lot of comments here but generally it looks good.

**Comment by ardan-bkennedy** [comment]
Going to close this since I have a new one.

**Pull request closed without merging**